.github/workflows/cherryPick.yml

```diff
@@ -79,31 +64,38 @@ jobs:
           USER: OSBotify
           TITLE_REGEX: Update version to ${{ env.NEW_VERSION }}
 
-      - name: Cherry-pick the version-bump to new branch
+      - name: Create branch for new pull request
         run: |
-          git fetch
-          git cherry-pick -S -x --mainline 1 --strategy=recursive -Xtheirs ${{ steps.getVersionBumpMergeCommit.outputs.MERGE_COMMIT_SHA }}
+          SOURCE_MERGE_BASE=$(git merge-base staging ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }})
+          VERSION_BUMP_MERGE_BASE=$(git merge-base staging ${{ steps.getVersionBumpMergeCommit.outputs.MERGE_COMMIT_SHA }})
+          CP_MERGE_BASE=$(git merge-base "$SOURCE_MERGE_BASE" "$VERSION_BUMP_MERGE_BASE")
+          git checkout -b ${{ github.actor }}-cherry-pick-staging-${{ github.event.inputs.PULL_REQUEST_NUMBER }} "$CP_MERGE_BASE"
+          git push --set-upstream origin ${{ github.actor }}-cherry-pick-staging-${{ github.event.inputs.PULL_REQUEST_NUMBER }}
 
-      - name: Cherry-pick the merge commit of target PR to new branch
-        id: cherryPick
-        run: |
-          echo "Attempting to cherry-pick ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }}"
-          if git cherry-pick -S -x --mainline 1 ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }}; then
-            echo "🎉 No conflicts! CP was a success, PR can be automerged 🎉"
-            echo "::set-output name=SHOULD_AUTOMERGE::true"
-          else
-            echo "😞 PR can't be automerged, there are merge conflicts in the following files:"
-            git --no-pager diff --name-only --diff-filter=U
-            git add .
-            GIT_MERGE_AUTOEDIT=no git cherry-pick --continue
-            echo "::set-output name=SHOULD_AUTOMERGE::false"
-          fi
+      - name: Cherry-pick the version-bump to new branch
+        run: git cherry-pick -S -x --mainline 1 -Xtheirs ${{ steps.getVersionBumpMergeCommit.outputs.MERGE_COMMIT_SHA }}
+
+      - name: Cherry-pick the merge commit of target PR to the new branch
+        run: git cherry-pick -S -x --mainline 1 ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }}
```

Comment:
Need to dig in to confirm, but I think there won't be conflicts against the temporary branch created from the `merge-base` of the two commits we're trying to CP.

I think the merge conflicts will pop up when we're attempting to merge code into staging (and even then, only if there are multiple conflicting CPs)